sites/public/src/components/listings/search/ListingsSearchModal.tsx

```diff
@@ -146,66 +176,82 @@ export function ListingsSearchModal(props: ListingsSearchModalProps) {
     <Modal
       open={props.open}
       onClose={props.onClose}
-      title={"Filters"}
+      title="Filters"
+      headerClassNames="bottom-border"
       ariaDescription="Listing Search Filters"
       actions={[
-        <button onClick={onSubmit}>Show matching listings</button>,
+        <Button type="button" className="is-secondary" onClick={onSubmit}>
+          Show matching listings
```

Comment:
All of the text in the buttons and labels should be translated, can you add it to general.json and call t() or do it as a follow up? My PR has it if you want to merge that but there are some extra labels in your PR

Reply:
Agreed. If you don't get to it today, I can get to it next week w/the translations, if needed.

Reply:
These translations will go into `doorway-ui-components/src/locales/general.json` or another general.json file?

Reply:
I'll resolve translations next week.